Rollbar/AspNetCore/RollbarLogger.cs

```diff
@@ -98,6 +98,15 @@ mslogging.LogLevel logLevel
             if (state == null && exception == null)
                 return;
 
+            if (RollbarScope.Current != null
+                && RollbarLocator.RollbarInstance.Config.MaxItems > 0
+                && RollbarScope.Current.LogItemsCount >= RollbarLocator.RollbarInstance.Config.MaxItems
```

Comment:
It looks like this will continually generate Warning items, not just one additional item. We don't want to send a max items reached warning instead of the actual message on every log call.

Reply:
@rokob, thanks a lot! that is a great catch! 
RollbarMiddleware had very similar problem. to make it even worse the IncrementLogItemsCount() was never called. looks like i somehow lost the final code state for this feature and ended up submitting one of the iterations instead of the final.
Thanks again for spotting this one!